samples/04.0-file-transfer/system-tests/README.md

```diff
@@ -0,0 +1,29 @@
+# File transfer system tests
+
+This system test serves to validate both the connector runtime and the EDC Helm chart with an end-to-end flow.
+It runs within GitHub Actions.
+
+To run it locally, install:
+- [Minikube and Kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [Helm](https://helm.sh/docs/intro/install/)
+- [Docker](https://docs.docker.com/get-docker/)
+
```

Comment:
For clarity maybe it would be worth adding 1 step more that consumer and provider jars must be built to run the test?

Build provider and consumer jars:
./gradlew samples:04.0-file-transfer:consumer:build
./gradlew samples:04.0-file-transfer:provider:build